src/ui/components/Explorer/Explorer.js

```diff
@@ -17,6 +17,7 @@ import {
 } from "@material-ui/core";
 import KeyboardArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
 import KeyboardArrowUpIcon from "@material-ui/icons/KeyboardArrowUp";
+import {StyleSheet, css} from "aphrodite/no-important";
```

Comment:
material ui has its own styled-component solution. Might want to use those instead to be consistent. 
https://material-ui.com/styles/basics/#styled-components-api

Reply:
This doesn't work with the explorer, bc the material-ui makeStyles only works with functional components. There is a HOC solution as well, but I had trouble getting it working with flow. I will change this on another branch where I am switching everything to hooks and functional components.

Reply:
The styled components API should work with components I believe.

Reply:
https://stackoverflow.com/questions/56432167/how-to-style-components-using-makestyles-and-still-have-lifecycle-methods-in-mat

If you do not use the HOC method, our ususal way of using makeStyles doesn't work inside of class components. This is bc makeStyles uses react hooks, and react forbids hooks within class components.

There's a way to use HOC instead of the normal makeStyles classes pattern, but I was having trouble getting this to work with flow. I could spend more time fiddling with this, but I will be converting this component to a functional component soon anyway, so I figure I will just use our standard pattern then.

Reply:
This is using styled. Should work with components.

```
Import { styled } from '@material-ui/core/styles'; 
import Button from '@material-ui/core/Button'; 
const MyButton = styled(Button)({ background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)', border: 0, borderRadius: 3, boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)', color: 'white', height: 48, padding: '0 30px', });
```